### ISSUE TITLE
Misrendered nested lists in 'Field Types' > 'Datetime Types' docs 

### ISSUE THREAD

**Issue by huonw** [samuelcolvin/pydantic#2455]
### Checks

* [x] I added a descriptive title to this issue
* [x] I have searched (google, github) for similar issues and couldn't find anything
* [x] I have read and followed [the docs](https://pydantic-docs.helpmanual.io/) and still think this is a bug

# Bug

Currently the online version of https://pydantic-docs.helpmanual.io/usage/types/#datetime-types is misrendered, where the nested lists have become text/code:

![image](https://user-images.githubusercontent.com/1203825/109756124-68513800-7c3b-11eb-948f-6ee8f4d13eca.png)

This renders correctly both on GitHub (https://github.com/samuelcolvin/pydantic/blob/master/docs/usage/types.md#datetime-types) and if I build the docs locally with `make docs`.

<details><summary>Click for my local dependency versions</summary>

```
ansi2html==1.6.0
appdirs==1.4.4
attrs==20.3.0
black==20.8b1
bleach==3.3.0
certifi==2020.12.5
chardet==4.0.0
click==7.1.2
colorama==0.4.4
coverage==5.4
Cython==0.29.21
devtools==0.6.1
dnspython==2.1.0
docutils==0.16
email-validator==1.1.2
flake8==3.8.4
flake8-quotes==3.2.0
future==0.18.2
hypothesis==5.44.0
idna==2.10
iniconfig==1.1.1
isort==5.7.0
Jinja2==2.11.3
joblib==1.0.1
keyring==22.0.1
livereload==2.6.3
lunr==0.5.8
Markdown==3.3.3
markdown-include==0.6.0
MarkupSafe==1.1.1
mccabe==0.6.1
mkdocs==1.1.2
mkdocs-exclude==1.0.2
mkdocs-material==6.2.8
mkdocs-material-extensions==1.0.1
mypy==0.800
mypy-extensions==0.4.3
nltk==3.5
orjson==3.4.8
packaging==20.9
pathspec==0.8.1
pkginfo==1.7.0
pluggy==0.13.1
py==1.10.0
pycodestyle==2.6.0
-e git+git@github.com:huonw/pydantic.git@ab6bef4ac81137a1e1802f4955cdb9b818954e6d#egg=pydantic
pyflakes==2.2.0
Pygments==2.8.0
pymdown-extensions==8.1.1
pyparsing==2.4.7
pytest==6.2.2
pytest-cov==2.11.1
pytest-mock==3.5.1
pytest-sugar==0.9.4
python-dotenv==0.15.0
PyYAML==5.4.1
readme-renderer==28.0
regex==2020.11.13
requests==2.25.1
requests-toolbelt==0.9.1
rfc3986==1.4.0
six==1.15.0
sortedcontainers==2.3.0
SQLAlchemy==1.3.23
termcolor==1.1.0
toml==0.10.2
tornado==6.1
tqdm==4.56.2
twine==3.3.0
typed-ast==1.4.2
typing-extensions==3.7.4.3
ujson==4.0.2
urllib3==1.26.3
webencodings==0.5.1
```

</details>

(Thanks for Pydantic!)

**Comment by PrettyWood** [comment]
Hi @huonw and thanks for reporting!
It has been fixed in #2448 :) this is why it works in local. Doc will be redeployed when 1.8.1 will be released (probably today or tomorrow)
Cheers!

**Comment by huonw** [comment]
Ah thanks! I missed that in my search.
